### PR TITLE
feat: student profile

### PR DESCRIPTION
**Pull request by 01001110J**
# Description

In this PR is added the profile section for students

This PR resolves [PADV-1177](https://agile-jira.pearson.com/browse/PADV-1177)

### How to test

Clone the Institution Portal MFE
```Bash
     git clone <url>
```
Install the packages 📦.
```Bash
     npm i
```
Start project.
```Bash
     npm start
```

Finally, go to http://localhost:1980/students/:username and see the results.

### CODE REVIEW

**Review by AuraAlba — CHANGES_REQUESTED** [review]

File: src/features/Main/index.jsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ const Main = () => {
   const routes = [
     { path: '/dashboard', component: DashboardPage, exact: true },
     { path: '/students', component: StudentsPage, exact: true },
+    { path: '/students/:learnerName', component: StudentsDetails, exact: true },
```
[the remark this answers is not in the document]
Comment (01001110J) [reply src/features/Main/index.jsx:42]: Good catch! I replaced the user name by user Email, in theory it should be unique.

[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Main/index.jsx]: Please check the cases when there are special characters, for example: student/@example.com is a valid email for the platform

File: src/features/Main/index.jsx [hunk 2]
```diff
@@ -38,6 +39,7 @@ const Main = () => {
   const routes = [
     { path: '/dashboard', component: DashboardPage, exact: true },
     { path: '/students', component: StudentsPage, exact: true },
+    { path: '/students/:learnerEmail', component: StudentsDetails, exact: true },
```
Comment (AuraAlba) [inline src/features/Main/index.jsx:42]: Please check the cases when there are special characters, for example: student/@example.com is a valid email for the platform

**Review by AuraAlba — CHANGES_REQUESTED** [review]

File: src/features/Main/index.jsx (hunk 2 above)
Comment (01001110J) [reply src/features/Main/index.jsx:42]: Fixed

**Review by anfbermudezme — APPROVED** [review]

**Pull request merged**